README.md

```diff
@@ -1,2 +1,140 @@
-# flat-image-tree
-Spec for the Flat Image Tree (FIT) specification
+# Document master project for the Devicetree Specification #
+
+The latest release of the specification can be found at http://devicetree.org/specifications/ or https://github.com/devicetree-org/devicetree-specification/releases
+
+This [repository](https://github.com/devicetree-org/devicetree-specification) holds the source for the generation of the Devicetree Specification using Sphinx and LaTeX. 
+
+## Mailing list: devicetree-spec@vger.kernel.org ##
+   * Use this mailing list for submitting patches, questions and general discussion
+   * Sign up to the mailing list at http://vger.kernel.org/vger-lists.html#devicetree-spec
```

Comment:
let me check with @walterchris  if we can set up a mailing list for FIT under OSFF.

Reply:
ok

Reply:
Hi Simon, will this works for you? :)
flat-image-tree@lists.osfw.foundation

Reply:
it is easy to subscribe:
https://lists.osfw.foundation/g/flat-image-tree

Reply:
Hmmm can we get rid of the elephants?

Reply:
I replace the 'elephants' with the FIT logo.

Reply:
thanks!